android-components/components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt

```diff
@@ -567,28 +604,6 @@ data class ContextMenuCandidate(
             )
         }
     }
-
```

Comment:
Moved SnackbarDelegate and DefaultSnackbarDelegate to SupportUtils

Reply:
To have it easily available from more places - eg the new `CopyDownloadFeature`, nice! 👍